From: Laurent Bigonville <bigon@debian.org>
Date: Fri, 4 Sep 2020 13:08:05 +0100
Subject: Disable the test-model-filter that requires a system bus

Forwarded: not-needed
---
 src/tests/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tests/meson.build b/src/tests/meson.build
index e2c1d65..d1b03b2 100644
--- a/src/tests/meson.build
+++ b/src/tests/meson.build
@@ -144,7 +144,7 @@ if get_option('enable_gtk')
     dependencies: test_ui_deps,
   )
 
-  test('test-model-filter', test_model_filter, env: test_env)
+#  test('test-model-filter', test_model_filter, env: test_env)
   test('test-zoom', test_zoom, env: test_env)
 
 endif
